vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/InputFieldView.kt

```diff
@@ -823,6 +823,8 @@ abstract class InputFieldView @JvmOverloads constructor(
     protected fun applyPreviewIconMode(mode: Int) {
         if (fieldType == FieldType.CARD_NUMBER) {
             (inputField as? CardInputField)?.setPreviewIconMode(mode)
+        } else if (fieldType == FieldType.CVC) {
+            (inputField as? CVCInputField)?.setPreviewIconVisibility(mode)
         }
```

Comment:
Could we use `when` construction here?

Reply:
@Endorf Actually I think `if/else if `better here because in case of `when` we will need to add extra else(and handle it somehow)